### PR TITLE
[CARBONDATA-740] add logger for total rows processed

### PR DESCRIPTION
**Pull request by lionelcao**
add logger for total rows processed

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1025/

File: processing/src/main/java/org/apache/carbondata/processing/newflow/AbstractDataLoadProcessorStep.java [hunk 1]
```diff
@@ -157,6 +157,7 @@ protected CarbonRowBatch processRowBatch(CarbonRowBatch rowBatch) {
   public void close() {
     if (!closed) {
       closed = true;
+      LOGGER.info("Total rows processed in step: " + rowCounter.get());
```
Comment (ravipesala) [inline processing/src/main/java/org/apache/carbondata/processing/newflow/AbstractDataLoadProcessorStep.java]: Please add the step name as well here.

**Comment by lionelcao** [comment]
@ravipesala  Added step name, please check.

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1032/

**Comment by ravipesala** [comment]
LGTM Thanks for working on this.

**Comment by lionelcao** [comment]
@ravipesala this PR is closed but not merged yet. I can't reopen it.

**Pull request closed without merging**